compiler/src/template-api/addUSMap.js

```diff
@@ -5,6 +5,14 @@ const Layer = require("../Layer").Layer;
 const Transform = require("../Transform").Transform;
 const getBodyStringOfFunction = require("./Utilities").getBodyStringOfFunction;
 
+const updateStateQuery = `SELECT cs.name, cs.state_id, cs.total_dem_votes, cs.total_rep_votes,
```

Comment:
These seem to be hardcoded for the election dataset. 

I think for simplicity, maybe we can make the usmap one in a separate PR? since it involves editing another set of APIs (the template APIs).